package.json

```diff
@@ -30,27 +30,36 @@
   "homepage": "http://fluidproject.org",
   "dependencies": {
     "es6-template-strings": "2.0.0",
-    "extend": "^3.0.0"
+    "extend": "3.0.0"
   },
   "devDependencies": {
-    "grunt": "^0.4.5",
-    "grunt-contrib-jshint": "~0.9.0",
+    "grunt": "1.0.1",
+    "eslint-config-fluid": "1.0.0",
+    "fluid-grunt-eslint": "18.1.2",
     "grunt-jsonlint": "1.0.4",
-    "sinon": "^1.17.1"
+    "sinon": "1.17.1"
   },
   "defaultOptions": {
```

Comment:
Note for the future - this strategy of packing configuration options into package.json is cute, but doesn't actually enable reuse of the functionality. In order to customise these strings, the user has to fork this module and change them! When we start work on the "mature multirepo fluid-publish" sketched out in #7, we should move this module over to being a standard Infusion component and have all these options as part of its defaults.